docs/src/statmodels.md

```diff
@@ -15,22 +15,32 @@ coeftable
 confint
 deviance
 dof
+fisherinf
```

Comment:
The suffix "inf" here is confusing. I assume you mean Fisher information, but "inf" makes it sound like a reference to `Inf`.

Reply:
What about `fisherim` as in information matrix or `fisherinformation`? I believe these methods will mostly be used by other packages and not user so we could have longer than usual names.

Reply:
Is there any precedent in other software? I could only find mentions of fields or functions called `hessian`.

Reply:
Stata has uses it as options for variance covariance estimations. It uses `eim` for expected and `oim` for the observed. Typically the `score` and `fisherinf` are used to compute the variance covariance which is what is typically returned. I would be fine not exporting those too to encourage their use through other packages that compute the variance covariance.

Reply:
OK. The situation is a bit different in Stata since it's an option, so short names do not polluted the namespace.

`fisherinformation` isn't too bad, but `observedfisherinformation` would be really too long. Yet having `fisherinformation` and `observedinformation` would be weird too, since both refer to Fisher. Maybe `informationmatrix(obj, kind::Symbol)` with `kind` being either `:observed` or `:expected`?

Reply:
I believe Fisher is the expected of the observed information matrix. However, I think having these and selecting on kind is preferred. How about,
```
informationmatrix(model::StatisticalModel; expected::Bool=true)
```

Reply:
Yeah, that's another possibility. Not sure the argument should be called `expected` or `observed` (disregarding the default, which I agree should be expected).

Reply:
How about `fisher::Bool=true` as Fisher indicates the expected rather than the observed?

Reply:
I think `expected` is a better name for the argument. We can document that `expected=true` indicates that the result is the Fisher information.

Reply:
Latest version reflects those changes.